rust_src/src/character.rs

```diff
@@ -0,0 +1,20 @@
+use std::os::raw::c_char;
+use std::ptr;
+
+use lisp;
+
+extern crate libc;
+
+use lisp::{LispObject, LispSubr, EmacsInt};
+
+/// Maximum character code ((1 << CHARACTERBITS) - 1).
+pub const MaxChar: EmacsInt = 0x3FFFFF;
+
+fn Fmax_char() -> LispObject {
+    lisp::make_number(MaxChar)
+}
+
+defun!("max-char", Fmax_char, Smax_char, 0, 0, ptr::null(),
+"Return the character of the maximum code.
+
+(fn)");
```

Comment:
Minor thing, but you don't need to declare the calling convention here. There are no arguments that we want to name.